crates/hc/README.md

```diff
@@ -2,44 +2,30 @@
 
 Provides the `hc` binary, a helpful CLI tool for working with Holochain.
 
-```shell
-$ hc -h
-
-holochain_cli 0.1.0
-Holochain CLI
-
-Work with DNA and hApp bundle files, set up sandbox environments for testing and development purposes, make direct admin
-calls to running conductors, and more.
+## Docs
 
-USAGE:
-    hc <SUBCOMMAND>
+This command gives you a suite of tools for developing, inspecting, executing, and testing your Holochain apps. Some top-level subcommands are implemented as separate crates, and others are separate binaries -- commands whose names start with `hc-` and are automatically made available as subcommands if they exist in your shell's path. Here is a list of all available subcommands:
 
-FLAGS:
-    -h, --help       Prints help information
-    -V, --version    Prints version information
+- `hc dna`, `hc app`, and `hc web-app` scaffold, bundle, and unbundle DNAs, hApps and web hApps respectively. See [holochain_cli_bundle](https://github.com/holochain/holochain/tree/develop/crates/hc_bundle) for more info.
+- `hc sandbox` creates and executes temporary or persistent conductor configurations for you to run test instances of your hApp with. See [holochain_cli_sandbox](https://github.com/holochain/holochain/tree/develop/crates/hc_sandbox) for more info.
+- `hc signal-srv` runs a local WebRTC signal server for peers to establish connections with each other. See [holochain_cli_signal_srv](https://github.com/holochain/holochain/tree/develop/crates/hc_signal_srv) for more info.
```

Comment:
This doesn't exist anymore. It's been renamed to `run-local-services`.

Reply:
dang, missed that in the text. Thanks.